tests/algorithms/test_preference_comparisons.py

```diff
@@ -966,74 +964,73 @@ def action_is_reward_agent(action_is_reward_venv, rng):
     )
 
 
-@pytest.fixture
-def action_is_reward_trainer(
-    action_is_reward_venv,
+def basic_reward_trainer(
```

Comment:
Nit: this probably doesn't need to be spread over multiple lines now there's only a single parameter. (black won't put things back together on one line once they have a trailing comma, but will happily split them up when they're too long.)

Reply:
Sure, done